packages/clients/diplan/example/config.js

```
@@ -59,10 +62,18 @@ export default {
     },
     {
       id: xplanwfs,
-      visibility: true,
+      visibility: false,
```

`hideInMenu` seems fitting in this case

---

I'm just turning it off by default and have no intention of hiding it right now. Maybe I'll have one later? I just don't know yet.

---

Oh, yea, right, a user can just turn it on when they have the client open 🤦 🎩